config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Portrait::Application.routes.draw do
-  resources :sites
-  resources :users
+  resources :customers, except: [:new, :edit]
```

Comment:
We are not using the full resource routes here, so limit them to the ones we are actually using. 

For example on sites, we only using index, create (don't need routes generated for show, new, edit, update, destroy)